src/Simplex/Messaging/Agent.hs

```diff
@@ -880,6 +881,8 @@ joinConnSrvAsync _c _userId _connId _enableNtfs (CRContactUri _) _cInfo _subMode
 createReplyQueue :: AgentClient -> ConnData -> SndQueue -> SubscriptionMode -> SMPServerWithAuth -> AM SMPQueueInfo
 createReplyQueue c ConnData {userId, connId, enableNtfs} SndQueue {smpClientVersion} subMode srv = do
   (rq, qUri, tSess, sessId) <- newRcvQueue c userId connId srv (versionToRange smpClientVersion) subMode
+  let RcvQueue {server} = rq
+  atomically $ incSMPServerStat c userId server connCreated
```

Comment:
this doesn't seem correct

Reply:
I think it is correct, we count creation of your own rcv queue as connection creation, otherwise connection creation stats are all over the place (created, secured, completed)